bigquery_etl/public_data/publish_gcs_metadata.py

```diff
@@ -51,21 +51,25 @@
 class GcsTableMetadata:
     """Metadata associated with table data stored on GCS."""
 
-    def __init__(self, files, endpoint, target_dir):
+    def __init__(self, blobs, endpoint, target_dir):
         """Initialize container for metadata of a table published on GCS."""
-        assert len(files) > 0
-        self.files = files
+        assert len(blobs) > 0
+        self.blobs = blobs
         self.endpoint = endpoint
-        self.files_path = self.files[0].split("files")[0] + "files"
+        self.files_path = self.blobs[0].name.split("files")[0] + "files"
         self.files_uri = endpoint + self.files_path
 
-        (self.dataset, self.table, self.version) = dataset_table_version_from_gcs_path(
-            self.files[0]
+        (self.dataset, self.table, self.version) = dataset_table_version_from_gcs_blob(
+            self.blobs[0]
         )
         self.metadata = Metadata.of_table(
             self.dataset, self.table, self.version, target_dir
         )
 
+        self.last_updated_path = self.blobs[0].name.split("files")[0] + "last_updated"
+        self.last_updated_uri = endpoint + self.last_updated_path
+        self.last_updated = min(self.blobs, key=lambda b: b.updated).updated
```

Comment:
```suggestion
        self.last_updated = min(b.updated for b in blobs)
```

Would this be equivalent to what you have written here?